cluster-logging/overlays/moc/zero/clusterlogforwarders/clusterlogforwarder.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
```

Comment:
Can this be named to something less generic? like `kafka-log-forwarder` maybe?

Reply:
This is required to be `instance` ([docs](https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-kafka_cluster-logging-external))

Reply:
Nevermind I see that it has to be `instance`:

From the docs: 
> The namespace for the ClusterLogForwarder CR must be openshift-logging.